kubectl/README.md

```diff
@@ -34,17 +49,31 @@ spec:
   taskRef:
     name: kubectl-deploy-pod
   params:
+  - name: action
+    value: create
+  - name: success-condition
+    value: status.phase == Running
+  - name: failure-condition
+    value: status.phase in (Failed, Error)
+  - name: output
+    value: |
+      - name: job-name
+        valueFrom: '{.metadata.name}'
+      - name: job-namespace
+        valueFrom: '{.metadata.namespace}' 
+  - name: set-ownerreference
+    value: "true"
   - name: manifest
     value: |
       apiVersion: v1
       kind: Pod
       metadata:
-        name: myapp-pod
+        generateName: myapp-pod-
         labels:
           app: myapp
       spec:
         containers:
         - name: myapp-container
           image: docker
-          command: ['sh', '-c', 'echo Hello Kubernetes! && sleep 3600']
+          command: ['sh', '-c', 'echo Hello Kubernetes! && sleep 30']
```

Comment:
Can we mount the `MANIFESTs` via the `ConfigMap` and `Workspaces` instead of specifying them over here in the `run`?  This will make the code cleaner and also we can mount many manifests at the same time eg mounting `deployment.yaml` and `service.yaml`. WDYT?
Will it satisfy your usecase also?
cc @savitaashture

Reply:
@vinamra28 
Thanks for your comments, 
I'm afraid for now can not support multiple manifest in one step.
The manifest will be created success, but the monitor/trace will failed, is that a common requirement?

Reply:
Okay...No issues.
Actually I was thinking to add a task which can support `kubectl` commands which may or may not require manifests eg `kubectl rollout status deployment.v1.apps/nginx-deployment`  or if in case we provide manifests via the workspaces then we are able to `apply` that also.

Reply:
Seems the version before this one could satisfy you 😸

Reply:
The previous version could only satisfy one of the scenario of applying multiple manifests by adding workspaces :sweat_smile: , one more support I want to add is to perform `kubectl` commands even if there is no manifest provided. (eg `kubectl get pods`). A kind of very generic task which can perform k8s actions :smile: . I'll try raising a PR for it. :smiley: